tests/bundle/Routing/ExpressionLanguage/ContentTypeHeaderMatcherFactoryTest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Tests\Bundle\Rest\Routing\ExpressionLanguage;
+
+use Closure;
+use Ibexa\Bundle\Rest\Routing\ExpressionLanguage\ContentTypeHeaderMatcherFactory;
+use Ibexa\Contracts\Rest\Input\ParsingDispatcher;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\HttpFoundation\Request;
+
+final class ContentTypeHeaderMatcherFactoryTest extends TestCase
+{
+    private readonly ParsingDispatcher&MockObject $parsingDispatcher;
+
+    private readonly Closure $contentTypeHeaderMatcher;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
```

Comment:
Parent `setUp()` method is empty. Most code I've seen that relates directly to PHPUnit recommends to omit it.
```suggestion
```